tests/components/create_post/create_post.test.jsx

```diff
@@ -699,4 +699,92 @@ describe('components/create_post', () => {
         const wrapper = shallow(createPost({canUploadFiles: false}));
         expect(wrapper).toMatchSnapshot();
     });
+
+    it('should allow to force send invalid slash command as a message', async () => {
+        const error = {
+            message: 'No command found',
+            server_error_id: 'api.command.execute_command.not_found.app_error',
+        };
+        const executeCommand = jest.fn(() => Promise.resolve({error}));
+        const onSubmitPost = jest.fn();
+
+        const wrapper = shallow(
+            createPost({
+                actions: {
+                    ...actionsProp,
+                    executeCommand,
+                    onSubmitPost,
+                },
+            })
+        );
+
+        wrapper.setState({
+            message: '/fakecommand some text',
+        });
+        expect(wrapper.find('[id="create_post.invalidCommand"]').exists()).toBe(false);
+
+        wrapper.instance().handleSubmit({preventDefault: jest.fn()});
+        expect(executeCommand).toHaveBeenCalled();
+
+        await Promise.resolve();
```

Comment:
I'd recommend to remove all manual call to resolve a promise.
```suggestion
 
```

Instead, you may change it into,
```javascript
await wrapper.instance().handleSubmit({preventDefault: jest.fn()});
```

Reply:
Great thanks for the tip!